.github/workflows/periodic.yml

```diff
@@ -3,6 +3,7 @@ name: periodic
 on:
   schedule:
     - cron: 45 0,4,8,12,16,20 * * *
+    - cron: 29 8 * * *  # about 1:29am PDT
```

Comment:
As the Windows mem_leak_check jobs would likely still timeout, I assume that you would still want https://github.com/pytorch/pytorch/pull/89455 merged, or set the `timeout-minutes` value here, right?

Reply:
ill just merge the other one first